torch/csrc/jit/tensorexpr/ir_printer.cpp

```diff
@@ -473,6 +473,11 @@ void IRPrinter::visit(FreePtr v) {
   os() << "Free(" << *v->buffer_var() << ");";
 }
 
+void IRPrinter::visit(PlacementAllocatePtr v) {
+  os() << "Alias(" << *v->buf()->base_handle() << ","
+       << *v->buf_to_reuse()->base_handle() << ");";
```

Comment:
Nit: it might be easier to comprehend which buffer reuses which if we printed it as `Alias old = new` or `Alias(old=$name_old, new=$name_new)`. If you prefer to keep it as is, please add a space after the comma.

Reply:
Bump :)